### PR TITLE
Fixed some presets and added some quest unlocks

### PR DESCRIPTION
**Pull request by Razzmatazzz**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /node_modules
-package-lock.json
\ No newline at end of file
+package-lock.json
+index.js
```
Comment (kokarn) [inline .gitignore:3]: This seems redundant?

**Review by kokarn — CHANGES_REQUESTED** [review]
I don't think the change to .gitignore should be here, right?

File: .gitignore (hunk 1 above)
Comment (Razzmatazzz) [reply .gitignore]: I sometimes add an index.js to work with the data and it's not really something we want in the repo.

**Comment by Razzmatazzz** [comment]
I don't think we want my personal data manipulation script in the repo, which is why I added that line to .gitignore.

**Comment by kokarn** [comment]
That's totally understandable. However, you can just keep that change locally and not push to GitHub.

It would be a bit of a head-scratcher at least to me to figure out why a newly added file locally (index.js) wasn't being picked up by git as a new file.

**Comment by Razzmatazzz** [comment]
Keeping a file locally and not pushing it to github is beyond my knowledge, so I just deleted it.

**Review by kokarn — APPROVED** [review]

**Pull request merged**